conans/test/integration/toolchains/microsoft/test_nmakedeps.py

```diff
@@ -38,8 +41,11 @@ def package_info(self):
     bat_file = client.load("conannmakedeps.bat")
     # Checking that defines are added to CL
     for flag in (
-        r"/DTEST_DEFINITION1", r"/DTEST_DEFINITION2#0",
-        r"/DTEST_DEFINITION3#", r'/DTEST_DEFINITION4#\\"foo\\"',
+        r'/D\"TEST_DEFINITION1\"', r'/D\"TEST_DEFINITION2#0\"',
+        r'/D\"TEST_DEFINITION3#\"', r'/D\"TEST_DEFINITION4#\"foo\"\"',
+        r'/D\"TEST_DEFINITION5#\"__declspec\(dllexport\)\"\"',
+        r'/D\"TEST_DEFINITION6#\"foo bar\"\"',
+        r'/D\"TEST_DEFINITION7#7\"'
```

Comment:
Can't we remove the ``r`` and the escaping backslashes? They seem unnecessary in most cases except ``/D\"TEST_DEFINITION6#\"foo bar\"\"`` (which by the way, it is the surprising thing, I am very surprised that nesting regular quotes works well.

Reply:
done!